### DISCUSSION TITLE
Why does the warp usd output not play at the fps set?

### DISCUSSION THREAD

**Discussion by cadop** [NVIDIA/warp#12] [Q&A] [answered]
When I set the warp simulation to a high frame per sec, like 1200 fps, and then load the USD file in omniverse, the playback looks very choppy instead of the high framerate/slow-motion.

**Comment by cadop** [comment, marked as answer]
So the answer seems to be a "omniverse create" issue/setting.  Warp _will_ honor the 1200fps setting, it isn't skipping anything.  Every frame is stored in USD and is set as 1fps.  However, if the computer can't render in time, it will skip the frame.  One way that seems to help visualize if the computer is slow is to set "Time Codes Per Second" (Under Layer Property of Create) to a value smaller than 1 (e.g. 0.1). 

Basically, as I understand it, warp acts as a traditional physics simulator, and Create is acting more like Unity or real-time game engine.
